moc_openshift.py

```diff
@@ -113,7 +117,7 @@ def user_exists(self, user_name):
             return True
         return False
 
-    def useridentitymapping_exists(self, user_name, id_provider, id_user):
+    def useridentitymapping_exists(self, user_name, id_user):
         user = self.get_user(user_name)
         if not (user.status_code in (200, 201)) and user["identities"]:
             id_str = f"{id_provider}:{id_user}"
```

Comment:
Here `id_provider` is undefined, since it's been removed from the function signature. There are a few other problems like this. Running the pre-commit checks locally before pushing changes will highlight these errors before they become part of the PR.

Reply:
I thought I had gotten that one.

Reply:
I think this should now be good.  Have also added the pylint check to my git pre-commit hook.